pkg/util/containers/providers/windows/provider.go

```diff
@@ -144,8 +144,23 @@ func (mp *provider) GetContainerMetrics(containerID string) (*metrics.ContainerM
 
 // GetContainerLimits returns CPU, Thread and Memory limits
 func (mp *provider) GetContainerLimits(containerID string) (*metrics.ContainerLimits, error) {
-	// FIXME: Figure out a way to extract limits from Job Objects in the Windows Kernel
-	return nil, fmt.Errorf("not supported on windows")
+	dockerUtil, err := docker.GetDockerUtil()
+	if err != nil {
+		return nil, err
+	}
+
+	cjson, err := dockerUtil.Inspect(containerID, false)
+	if err != nil {
+		return nil, err
+	}
+
+	containerLimits := metrics.ContainerLimits{
+		CPULimit: float64(cjson.HostConfig.NanoCPUs),
```

Comment:
I checked what we report on Linux and it's a percentage 0-100. So I believe we should get the number of CPUs from a syscall or if not possible, we can get it from Docker.

Reply:
So, here what I have when testing:
- `CPUShares` is a relative weight, not really a limit per se I'd say.
- `CPUCount` is not bounded, it can be set to `20` on `2 CPUs` machine but can be used to compute a limit.
- `CPUPercent` should is 1-100 as expected and can be used to compute a limit.

From https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/resource-controls
`CPUCount` and `CPUPercent` maps to `JOB_OBJECT_CPU_RATE_CONTROL_HARD_CAP`, which is what we're interested in.

On top of that the `NanoCPUs` is also used and seems to override the `CPUPercent`. Basically we need to reproduce the code computing `cpuMaximum` from:
https://github.com/moby/moby/blob/10866714412aea1bb587d1ad14b2ce1ba4cf4308/daemon/oci_windows.go#L426

I checked on GKE and in fact currently CPU Limit is not properly handled by K8S, it should be handled better with this PR:
https://github.com/moby/moby/blob/10866714412aea1bb587d1ad14b2ce1ba4cf4308/daemon/oci_windows.go#L426

Basically, the order to check limits for us should be:
1. `NanoCPUs`
2. `CPUPercent`
3. `CPUCount`